plugins/woo-ai/src/image-background-removal/background-removal-link.tsx

```diff
@@ -29,15 +30,23 @@ const getErrorMessage = ( errorCode?: string ) => {
 	}
 };
 
+const recordTracks = recordTracksFactory( 'image_background_removal', () => ( {
+	post_id: getPostId(),
+} ) );
+
 export const BackgroundRemovalLink = () => {
 	const { fetchImage } = useBackgroundRemoval();
 	const [ state, setState ] = useState< '' | 'generating' | 'uploading' >(
 		''
 	);
 	const [ displayError, setDisplayError ] = useState< string | null >( null );
 
+	recordTracks( 'view_ui' );
```

Comment:
This is running many times (with each render). We may want to throw it in a `useEffect` to cut down on the noise.